snippets/dart_snippets/lib/service_status.dart

```diff
@@ -3,7 +3,7 @@ import 'package:breez_sdk/bridge_generated.dart';
 
 Future<void> healthCheckStatus() async {
   // ANCHOR: health-check-status
-  ServiceHealthCheckResponse healthCheck = await BreezSDK().serviceHealthCheck();
+  ServiceHealthCheckResponse healthCheck = await BreezSDK().serviceHealthCheck("<api key>");
```

Comment:
Missing named parameter:
```suggestion
  ServiceHealthCheckResponse healthCheck = await BreezSDK().serviceHealthCheck(apiKey: "<api key>");
```